docs/deployment/README.md

```diff
@@ -39,18 +39,34 @@ The process involves:
 
 #### Install Terraform
 
-If you are using a Mac and [Homebrew](https://brew.sh), installing Terraform is
+It is recommended to keep a consistent version of Terraform as you deploy. Once your state files are migrated to a higher version, they are not always backwards compatible so integrators should pin their Terraform version. This is easily accomplished using the Terraform Version Manager [tfenv](https://github.com/tfutils/tfenv).
+
+If you are using a Mac and [Homebrew](https://brew.sh), installing tfenv is
 as simple as:
 
 ```shell
 brew update
-brew install terraform
+brew install tfenv
 ```
 
 For other cases,
-[installation instructions](https://learn.hashicorp.com/terraform/getting-started/install.html)
+[installation instructions](https://github.com/tfutils/tfenv#installation)
 are available.
 
+```shell
+$ tfenv install 0.12.12
+[INFO] Installing Terraform v0.12.13
+...
+[INFO] Switching completed
+
+$ tfenv use 0.12.12
+[INFO] Switching to v0.12.12
+...
+[INFO] Switching completed
+```
+
+It is recommended to stay on the Cumulus Core TF version which can be found [here](https://github.com/nasa/cumulus/blob/master/example/.tfversion). Any changes to that will be noted in the release notes.
```

Comment:
We can probably remove lines 70 - 75 with these new instructions to use tfenv?

Reply:
I left some of it because I think it's more clear to see what version you're _actually_ using with the `terraform` command. You can do it with `tfenv`, but it lists all your versions and puts a star next to one.